### PR TITLE
Homework3: Yuliya Golovach

### PR DESCRIPTION
**Pull request by YuliyaGolovach**

### CODE REVIEW

**Review by belovmd — CHANGES_REQUESTED** [review]
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-python-spring/blob/master/homework_review/Homework3.md):
- 2 
- 7
- 8
- 9

**Comment by YuliyaGolovach** [comment]
I know that attentiveness in not my strength, but sometime i really get surprised how i could wrote this (this code), for example wrote smth. that solves a different task...

File: src/hw_3/t03_tuple_practice.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+tuple2 = ('a', 'b', 'c')
+list2 = list(tuple2)
+print(type(list2), list2)
+
+
+a, b, c = 'a', 2, 'python'
+print("a = {a}, b = {b}, c = {c}".format(a=a, b=b, c=c))
+
+
+tuple4 = tuple([[1, 2, 3], ], )
```
Comment (belovmd) [inline src/hw_3/t03_tuple_practice.py:25]: tuple([1,2,3],)

**Review by belovmd — COMMENTED** [review]
9

**Pull request closed without merging**